Updated references to the documentation and releases on the website (#644)
strimzi/strimzi-kafka-bridge: README.md

## Documentation

Documentation can be found on our website:

> * for the [latest stable release](https://strimzi.io/docs/bridge/latest/)
> * for the "in development" [_main_ branch](https://strimzi.io/docs/bridge/in-development/)

Reviewer: ```suggestion
* [Documentation for the latest stable release](https://strimzi.io/docs/bridge/latest/)
* [Documentation that's _in development_ for the `main` branch](https://strimzi.io/docs/bridge/in-development/)
```

Reviewer: Not for me. I think the link text for both describes what we're linking to. Maybe if we were saying more we wouldn't want to have a full sentence link, for example.

> All releases are also available in the [downloads](https://strimzi.io/downloads/) section.

Reviewer: ```suggestion
Documentation for previous releases is also available in the [downloads](https://strimzi.io/downloads/) section.
```

Author: it's not about documentation, they are actually the releases :-)

Reviewer: Ah, okay. I've changed the suggestion.

Author: Ok. I will just specify "Strimzi bridge releases".
